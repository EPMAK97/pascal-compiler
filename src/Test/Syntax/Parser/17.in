function sum(a, b : integer) : integer;
    function mul(a, b : integer) : double;
    Begin
        result := a * b;
    end;
Begin
    result := a + b;
end;
Begin

end.