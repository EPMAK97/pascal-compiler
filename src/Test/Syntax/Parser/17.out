└── Main Program
    └── function
        ├── integer
        │   ├── a
        │   └── b
        ├── integer
        ├── function
        │   ├── integer
        │   │   ├── a
        │   │   └── b
        │   ├── double
        │   └── result
        │       └── *
        │           ├── a
        │           └── b
        └── result
            └── +
                ├── a
                └── b
